src/main/webapp/app/account/password/password.component.html

```diff
@@ -1,92 +1,119 @@
-<div>
+<div *ngIf="isRegistrationEnabled">
     <div class="row justify-content-center">
-        <div class="col-md-8">
-            <h2 jhiTranslate="password.title" [translateValues]="{ username: user.login }" *ngIf="user">
-                Password for [<b>{{ user.login }}</b
+        <div class="col-md-8" *ngIf="user">
+            <h2 jhiTranslate="password.title" [translateValues]="{ username: user.login }">
+                Password for [<strong>{{ user.login }}</strong
```

Comment:
What is the reason for the `[`/`]` brackets? I find them quite uncommon in this context.

Reply:
fixed